prov/efa/src/efa_prov_info.c

```diff
@@ -64,13 +64,43 @@
 
 #define EFA_NO_DEFAULT -1
```

Comment:
This commit has no commit message, could you add it?

Reply:
Added in new revision.